src/core/index.ts

```diff
@@ -1,29 +1,35 @@
-export * from './model/container';
-export * from './model/containerInstance';
-export * from './model/deploymentElement';
-export * from './model/deploymentNode';
-export * from './model/element';
-export * from './model/httpHealthCheck';
-export * from './model/iequatable';
-export * from './model/interactionStyle';
-export * from './model/location';
-export * from './model/model';
-export * from './model/modelItem';
-export * from './model/person';
-export * from './model/relationship';
-export * from './model/relationships';
-export * from './model/softwareSystem';
-export * from './model/staticStructureElement';
-export * from './model/tags';
-export * from './model/workspace';
+export * from "./model/container";
```

Comment:
This file was all single-quoted, I converted to double quotes for consistency as I was adding some exports. Let me know if not in scope and I'll revert.

Reply:
This is fine, thanks!